### PR TITLE
[FIX] - Metadata duplication task

### PR DESCRIPTION
**Pull request by AyakorK**
## Description

This PR is made with the objective to fix one task that didn't work at all before this. 
The initial objective of this task was to duplicate metadata from authorization handlers using a rake task.
It seemed that failing tests weren't notified coreclty and that the feature didn't work for a while and even probably since its creation. This PR was made in cooperation with @Quentinchampenois 

## What changed ?

- We separated the task in two ways
  - One task file to execute it
  - A class that will contain what we wanted about this feature
- We clarified and add some new specs

### CODE REVIEW

**Comment by Quentinchampenois** [comment]
We should separate your PR subject (metadata duplication task) and other fixes (head.html.erb / locales fixes / CI / etc).

What I suggest is to create new branch with commits before `e2b0ff0` and then change target of this PR on the newly created branch

File: lib/decidim/duplicates_metadata.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  class DuplicatesMetadata
+    def initialize
+      @authorizations = Decidim::Authorization.where(name: %w(data_authorization_handler extended_socio_demographic_authorization_handler))
+    end
+
+    def perform
+      update_authorized_users
+    end
+
+    def update_authorized_users
```
Comment (Quentinchampenois) [inline lib/decidim/duplicates_metadata.rb:13]: `updated_authorized_users` and `update_metadata` are not easy to read, we should consider refactor to keep it simple

File: spec/lib/decidim/duplicates_metadata_spec.rb [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+           name: "data_authorization_handler")
+  end
+  let!(:data_authorization_user2) do
+    create(:authorization,
+           user: user2,
+           metadata: { "minmum_age" => true, "gdpr" => true, "zipcode" => "54321" },
+           name: "data_authorization_handler")
+  end
+
+  context "when user has no metadata" do
```
Comment (Quentinchampenois) [inline spec/lib/decidim/duplicates_metadata_spec.rb:40]: It is easier if we use `shared_examples`  and `it_behaves_like` clausure to simplify reading and file

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
We must refactor the existing to simplify maintainance

**Review by armandfardeau — APPROVED** [review]

**Pull request merged**